middle_end/flambda/un_anf.ml

```diff
@@ -172,20 +195,28 @@ let make_var_info (clam : Clambda.ulambda) : var_info =
       loop ~depth ~environment_vars body;
       environment_vars
     | Uletrec (defs, body) ->
+    let environment_vars =
       List.fold_left (fun (var, def) ->
```

Comment:
This anonymous function needs to take the `environment_vars` as argument, like the others